CHANGELOG.md

```diff
@@ -3,7 +3,7 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## v1.1.2 2023-03-08
```

Comment:
Nit, maybe keep empty unreleased section in changelog to keep format

Reply:
This is only outside of releases, so we know what goes in next release. Check out other releases, they should not have a Unreleased group